src/libraries/System.IO.MemoryMappedFiles/src/System.IO.MemoryMappedFiles.csproj

```diff
@@ -7,157 +7,79 @@
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.FormatMessage.cs">
-      <Link>Common\Interop\Windows\Interop.FormatMessage.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.FileAttributes.cs">
-      <Link>Common\Interop\Windows\Interop.FileAttributes.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.FileOperations.cs">
-      <Link>Common\Interop\Windows\Interop.FileOperations.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.FlushViewOfFile.cs">
-      <Link>Common\Interop\Windows\Interop.FlushViewOfFile.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.GenericOperations.cs">
-      <Link>Common\Interop\Windows\Interop.GenericOperations.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.GetSystemInfo.cs">
-      <Link>Common\Interop\Windows\Interop.GetSystemInfo.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.SYSTEM_INFO.cs">
-      <Link>Common\Interop\Windows\Interop.SYSTEM_INFO.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.HandleOptions.cs">
-      <Link>Common\Interop\Windows\Interop.HandleOptions.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.MEMORY_BASIC_INFO.cs">
-      <Link>Common\Interop\Windows\Interop.MEMORY_BASIC_INFO.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.MEMORYSTATUSEX.cs">
-      <Link>Common\Interop\Windows\Interop.MEMORYSTATUSEX.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.MemOptions.cs">
-      <Link>Common\Interop\Windows\Interop.MemOptions.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.PipeOptions.cs">
-      <Link>Common\Interop\Windows\Interop.PipeOptions.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Windows\Interop.BOOL.cs">
-      <Link>Common\Interop\Windows\Interop.BOOL.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.SecurityOptions.cs">
-      <Link>Common\Interop\Windows\Interop.SecurityOptions.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.UnmapViewOfFile.cs">
-      <Link>Common\Interop\Windows\Interop.UnmapViewOfFile.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.VirtualQuery.cs">
-      <Link>Common\Interop\Windows\Interop.VirtualQuery.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)System\IO\Win32Marshal.cs">
-      <Link>Common\System\IO\Win32Marshal.cs</Link>
-    </Compile>
+    <Compile Include="$(CommonPath)Interop\Windows\Interop.BOOL.cs" Link="Common\Interop\Windows\Interop.BOOL.cs" />
```

Comment:
IMHO, this reformatting is questionable. It produces super long lines that are hard to read - see discussion in https://github.com/dotnet/runtime/issues/1266

Reply:
The reason why I did this is because it's easier to sort single lines alphabetically. Once the `<Link></Link>` is introduced, it gets formatted in a new line, making it difficult to sort.

But if the idea is to unify and avoid longer lines, I'll revert this change.